service/src/test/java/com/commercetools/pspadapter/payone/PaymentHandlerTest.java

```diff
@@ -271,6 +266,6 @@ private static CustomFields customFields(final String referenceValue) throws IOE
     }
 
     private static String randomString() {
-        return Integer.toString(random.nextInt());
+        return Integer.toString(random.nextInt(0x7FFFFFFF));
```

Comment:
default `nextInt` was too short and sometimes collisions in order numbers appeared in the tests